JDBC/OAuthBrowserFlowSampleApp/src/main/java/OAuthHandler.java

```diff
@@ -0,0 +1,134 @@
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.*;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+
+public class OAuthHandler {
+    // set these in the properties file to match your configuration
+    String connectionString = "";
+    String adminUser = "";
+    String adminUserPassword = "";
+    String oAuthUser = "";
+    String authName = "";
+    String clientId = "";
+    String clientSecret = "";
+    String httpsDiscoveryUrl = "";
+
+    Connection adminConnection;
+
+    public static void main(String[] args) {
+        try {
+            OAuthHandler oauthHandler = new OAuthHandler();
+            oauthHandler.doAuth();
+        } catch (Exception ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    public void doAuth() throws RuntimeException, URISyntaxException, IOException, SQLException {
```

Comment:
I like the way you structured the sample.   It allows people to easily skip over the setup code and go straight to the login example.